src/main/recorder.ts

```diff
@@ -280,6 +289,52 @@ type RecorderOptionsType = {
         obsRecorder.reconfigure(this._options);
         if (mainWindow) mainWindow.webContents.send('refreshState');
     }
+
+    /**
+     * Generate a filename for the final video of a recording according
+     * to its category, if possible.
+     *
+     * The final filename should contain adequate information to quickly be able
+     * to identify the video files on the filesystem and be able to tell what it
+     * was about and the result of it.
+     */
+    getFinalVideoFilename (metadata: Metadata): string | undefined {
+        let outputFilename = '';
+        let zoneName: string;
+        const resultText = getVideoResultText(metadata.category, metadata.result);
+
+        switch (metadata.category) {
+            case VideoCategory.TwoVTwo:
+            case VideoCategory.ThreeVThree:
+            case VideoCategory.Skirmish:
+            case VideoCategory.SoloShuffle:
+                zoneName = getInstanceNameByZoneId(metadata.zoneID);
+                outputFilename = `${zoneName} (${resultText})`;
```

Comment:
No resultText for SoloShuffles
Also BGs, but they are missed out of this function, I think by mistake?

Reply:
Most likely because I know nothing about PvP. I made a change now where, if `resultText` is empty it won't add it to the filename - will that satisfy SoloShuffles and BGs?

Reply:
Sounds good. The problem with solo shuffle is that there are 6 round and you can win any number of that. BGs there is just no indication in logs who wins. Currently in either case we don't display a result.